appmode/appmode_bottom.css

```diff
@@ -34,3 +34,7 @@
 #menubar-container {
     display: none !important;
 }
+
+#ipython_notebook {
+    flex-grow: 1 !important;
```

Comment:
Small note: This left-aligns the Jupyter logo and right-aligns buttons instead of somewhat centering and spacing them evenly across the header like the code currently does..